Frontend/ui-driver/src/Flow.purs

```diff
@@ -80,6 +80,7 @@ baseAppFlow baseFlow = do
     checkVersion versionCode
     cacheAppParameters versionCode
     when baseFlow $ void $ UI.splashScreen state.splashScreen
+    setValueToLocalNativeStore NEGOTIATION_UNIT if (getMerchant unit == YATRIPARTNER) then "20" else "10"
```

Comment:
do we need `NEGOTIATION_UNIT ` ?

Reply:
yes this is required for yatri partner. We are setting the value by which the driver can increment the price as 20 as we are taking it from shared prefs.